README.md

```diff
@@ -120,16 +130,6 @@ module.exports = {
 };
 ```
 
-## Updating .forceignore
-
-After adding Jest tests, you'll get errors trying to sync your local files with a scratch org because the `__tests__` directory will not be recognized. To ignore these test files, add this entry to your `.forceignore` file:
-
-```
-**/__tests__/**
-```
-
-See [How to Exclude Source When Syncing or Converting](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_exclude_source.htm) for more details.
-
 ## Resolving External Lightning Web Components
 
 Any Ligntning web components not located in your local `lwc` directory of your SFDX workspace will need to be mocked in your Jest tests. Included in this package are a set of mocks for all the `lightning` namespaced components.
```

Comment:
If a Lightning web component isn't located in the local `lwc` directory of your Salesforce DX project, you must mock it in your Jest tests. This package includes a set of mocks for all components in the `lightning` namespace.